python/src/aiconfig/editor/server/server_utils.py

```diff
@@ -501,3 +505,29 @@ def run_aiconfig_operation_with_request_json(
                 code=400,
                 aiconfig=None,
             ).to_flask_format()
+
+def validate_env_file_path(request_env_path: str | None | Any) -> Tuple[bool, str]:
+    """
+    Validates the given env file path. If its not valid, returns a tuple of (False, str) with a message.
+
+    Returns: (bool, str)
+    """
+    if request_env_path is None:
+        return False, "No .env file path provided"
+
+    if not isinstance(request_env_path, str):
+        return False, "Invalid request, specified .env file path is not a string"
```

Comment:
Nit, might be nicer to have `f"Invalid request: ${message}" above and not have request mentioned here